complaints/streamParser.py

```diff
@@ -59,6 +64,8 @@ def parse_json_file(input_file_name, output_file_name, logger):
                 elif (prefix, event) == ('data.item', 'end_array'):
                     new_complaint = dict(zip(my_column_array, my_data_array))
                     new_complaint["has_narrative"] = (not (not new_complaint["complaint_what_happened"] or len(new_complaint["complaint_what_happened"]) == 0))
+                    new_complaint["date_received_formatted"] = format_date(new_complaint["date_received"])
+                    new_complaint["date_sent_to_company_formatted"] = format_date(new_complaint["date_sent_to_company"])
```

Comment:
Do we think there will ever be a situation where one or both of these dates might be null or missing that we might want to handle?

Reply:
Right now it hasn't happened yet with the data, but it doesn't mean never.  That's a good suggestion, let me take a look at it.